msu/systems/mod_settings/abstract_setting.nut

```diff
@@ -160,25 +160,28 @@
 	}
 
 
-	function flagSerialize( _modID )
+	function flagSerialize()
 	{
-		this.setFlagForProperty("Value", _modID);
-		this.setFlagForProperty("Locked", _modID);
-		this.setFlagForProperty("LockReason"_modID);
+		local modID = this.getModID();
```

Comment:
I wouldn't bother with a local variable when the function is this short

Reply:
It's a single function call vs 3 function calls to a function which calls other functions, so why is it not preferable?